### ISSUE TITLE
🐛 Failing Run Go Unit Tests

### ISSUE THREAD

**Issue by ASTRobinson** [ministryofjustice/modernisation-platform#6387]
### Expected Behavior

Run Go Unit Tests run and complete successfully

Example here: https://github.com/ministryofjustice/modernisation-platform-github-oidc-role/actions/runs/8108945787

### Actual Behavior

Run Go Unit Tests is failing with 'ParameterNotFound' error. 

Example here: https://github.com/ministryofjustice/modernisation-platform-github-oidc-role/actions/runs/8142321400

### Steps to Reproduce the Problem

Create a new PR on the [modernisation-platform-github-oidc-role](https://github.com/ministryofjustice/modernisation-platform-github-oidc-role) repo.

also spotted the same error on following repositories:
[modernisation-platform-terraform-s3-bucket-replication-role](https://github.com/ministryofjustice/modernisation-platform-terraform-s3-bucket-replication-role/actions/runs/8144108779)
[modernisation-platform-terraform-pagerduty-integration](https://github.com/ministryofjustice/modernisation-platform-terraform-pagerduty-integration/actions/runs/8143967416)

**Comment by dms1981** [comment]
Having taken a quick look, it appears that the SSM parameter is missing from the account and that this bug isn't a problem related to the unit-test code in this repository. I'll do some `git` archaeology in https://github.com/ministryofjustice/modernisation-platform/terraform/environments/testing and see what I can find.

**Comment by dms1981** [comment]
By re-running the `member-bootstrap` code against `testing-test` I recreated the missing parameter. I could see that it was deleted in this CloudTrail Event in the `testing-test` account: https://eu-west-2.console.aws.amazon.com/cloudtrailv2/home?region=eu-west-2#/events/d685c8b6-cef8-4ac9-86f3-b2a2736b2b02
However, it looks like it was removed by a GitHub action in the `modernisation-platform-environments` repository - [aws-nuke](https://github.com/ministryofjustice/modernisation-platform-environments/actions/runs/8129979409).
It would have been recreated by our scheduled baseline job but I think we just got caught out here by the order of operations.
